docs/platform/webcontainers/roadmap.md

```diff
@@ -0,0 +1,36 @@
+---
+title: &title Roadmap
+description: &description "Get insights into our current and future plans for features in WebContainer. Let us know what you think!"
+head:
+  - ['meta', {property: 'og:title', content: *title}]
+  - ['meta', {property: 'og:image', content: 'https://developer.stackblitz.com/img/og/webcontainer-project-configuration.png'}]
+  - ['meta', {name: 'twitter:title', content: *title}]
+  - ['meta', {name: 'twitter:description', content: *description}]
+---
+
+# {{ $frontmatter.title }}
+
+:::warning Disclaimer
+
+The following is intended for informational purposes only and is subject to change. It might contain outdated information or inaccuracies.
+:::
+
+On this page, you'll find information about what is currently being worked on, planned features and things that are under consideration.
+
+If you have a feature request, please check [our github issue tracker](https://github.com/stackblitz/webcontainer-core/issues) to create a new issue or comment to an existing one.
+
+:::info
+This roadmap only includes large features. Minor bug fixes or features will not be reflected here.
+:::
+
+## 🚧 In Development
+
+ * **Node 18** (Github: https://github.com/stackblitz/webcontainer-core/issues/560)
+ * **WASI** (Github: https://github.com/stackblitz/webcontainer-core/issues/705)
+ * **Reducing memory usage** (Github: https://github.com/stackblitz/webcontainer-core/issues/1120)
+ * **Private / Custom Registries** (Github: https://github.com/stackblitz/webcontainer-core/issues/21)
+
+## 🦄 Future
```

Comment:
We can also mention `Supporting multiple Node.js versions`. I think it's something we can and should commit to.